icu4c/source/common/locid.cpp

```diff
@@ -509,6 +510,32 @@ Locale::operator==( const   Locale& other) const
 
 #define ISASCIIALPHA(c) (((c) >= 'a' && (c) <= 'z') || ((c) >= 'A' && (c) <= 'Z'))
 
+CharString& AppendLSCVE(CharString& out, const char* language, const char* script,
```

Comment:
It looks like this is a file-local helper function. Please enclose it in an anonymous namespace.